CMakeLists.txt

```diff
@@ -246,7 +249,7 @@ ENDIF (GTK3_FOUND)
 
 # Gnu Guile scheme interpreter
 # Version 2.2.2 is needed for mrs io ports, compilation, atomics, nlp
-FIND_PACKAGE(Guile 2.2.2)
+FIND_PACKAGE(Guile 2.0.2)
 IF (GUILE_FOUND AND GMP_LIBRARY AND GMP_INCLUDE_DIR)
```

Comment:
why are you down-grading guile? Like the comment up above says: `Version 2.2.2 is needed for mrs io ports, compilation, atomics, nlp`